plugin/mutate/source/dextool/plugin/mutate/frontend/argparser.d

```diff
@@ -454,6 +454,7 @@ struct ArgParser {
             int maxAlive = -1;
             int parallelMutants;
             long mutationTesterRuntime;
+            string metadataPath;
```

Comment:
you can remove this and instead directly set the variable because it is a string